_pytest/compat.py

```diff
@@ -253,6 +253,29 @@ def safe_str(v):
             return v.encode('ascii', errors)
 
 
+COLLECT_FAKEMODULE_ATTRIBUTES = (
+    'Collector',
+    'Module',
+    'Generator',
+    'Function',
+    'Instance',
+    'Session',
+    'Item',
+    'Class',
+    'File',
+    '_fillfuncargs',
+)
+
+
+def _setup_collect_fakemodule():
```

Comment:
I think it would be easier to just have a `collect.py` module which imports the symbols then, no? Less magic here would be better IMHO. Some tools might be confused/fail/warn that they can't find the file for `pytest.collect`, for example.